homeassistant/components/justnimbus/const.py

```diff
@@ -0,0 +1,13 @@
+"""Constants for the JustNimbus integration."""
+
+from typing import Final
+
+from homeassistant.const import Platform
+
+DOMAIN = "justnimbus"
+
+VOLUME_FLOW_RATE_LITERS_PER_MINUTE: Final = "L/m"
```

Comment:
`m` is used for `meters` according to SI standards, I think this needs to be

```suggestion
VOLUME_FLOW_RATE_LITERS_PER_MINUTE: Final = "L/min"
```